sdk/spring/spring-cloud-azure-appconfiguration-config/src/main/java/com/azure/spring/cloud/config/AppConfigurationRefresh.java

```diff
@@ -8,7 +8,7 @@
 import org.springframework.context.ApplicationEventPublisherAware;
 import org.springframework.scheduling.annotation.Async;
 
-import com.azure.spring.cloud.config.health.AppConfigurationStoreHealth;
+import com.azure.spring.cloud.config.implementation.health.AppConfigurationStoreHealth;
```

Comment:
line 41, the api exposes a class from the impl package, `AppConfigurationStoreHealth`.